src/themes/eui-amsterdam/overrides/_popover.scss

```diff
@@ -1,15 +1,17 @@
 .euiPopover__panel {
   .euiPopover__panelArrow {
     &:before {
-      filter: blur(3px);
+      filter: blur($euiSizeXS - 1px);
+      opacity: .2;
     }
 
+    $arrowShadowCompensation: $euiSizeXS;
```

Comment:
From what I could tell the `5px` values you had before were more than enough to reveal the shadow. We really want to use our sizing variables as much as possible, so I tested with our `XS` value (4px) and it also seems enough to reveal the shadow.

Reply:
👍 That's a good idea.